packages/framework/src/Framework/Features/Publications/Models/PublicationFieldType.php

```diff
@@ -29,27 +29,29 @@ class PublicationFieldType implements SerializableContract
     ];
 
     public readonly string $type;
-    public readonly ?int $max;
-    public readonly ?int $min;
+    public readonly string $max; // FIXME: needs to be string (dates, etc), should not be null, default = 0?
```

Comment:
Yeah, using strings definitely make sense. As for not being null, what if a user doesn't want length/range validation what would be the default value? `0` is a valid range, I had null to designate that the setting is not applied. What do you think makes most sense for this case?